RocketChat/Rocket.Chat.ReactNative
# Chore: Evaluate Message - TypeScript

## Proposed changes
Evaluate Message - TypeScript

## Issue(s)	


## How to test or reproduce
Test everything related to messages:
- not read messages label;
- date and time label;
- user name and avatar on the message; 
- tag another user;
- tag yourself;
- react to a message with an emoji:
	- check the emojis reacted on a message;
	- long press the emoji to open the reactions modal;
- quote a message and check if the quote message is working fine;
- edit a message;
- test old messages;
- test messages with images;
- test messages with attachments;
- test audios; 
- test messages threads;
- test messages discussions;
- test messages with link to other message;
- test messages with link to open in app; 
- test the read messages check (needs to activate on server)

## Screenshots

## Types of changes




## Checklist



## Further comments
